nano/node/socket.cpp

```diff
@@ -240,12 +240,17 @@ void nano::server_socket::on_connection (std::function<bool (std::shared_ptr<nan
 	boost::asio::post (strand, boost::asio::bind_executor (strand, [this_l, callback_a] () {
 		if (!this_l->acceptor.is_open ())
 		{
-			this_l->node.logger.try_log ("Network: Acceptor is not open");
+			this_l->node.logger.always_log ("Network: Acceptor is not open");
 			return;
 		}
+		this_l->evict_dead_connections ();
```

Comment:
This section is a copy and paste of the section below starting L265. Not sure if the duplicate if statement was needed? If it is then both blocks should be doing the same thing.

Reply:
I think we should remove this statement. Don't see why we need to check this twice.